Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -76,7 +77,6 @@ class BTPayPalNativeOrderCreationClient {
                     orderID: hermesResponse.orderID
                 )
 
-                self.apiClient.sendAnalyticsEvent("ios.paypal-native.create-order.succeeded")
```

Comment:
We don't need to send analytics in this file at all. Since we're already sending analytics in BTPayPalNativeCheckoutClient for the call to `orderCreationClient.createOrder` and switching on the `Result` type. 

The way it is now - there will be duplicate events.

Reply:
right now, we are only sending orderCreationFailed from the BTPayPalNativeOrderCreationClient. I did it this way because the unit tests were set up in the corresponding test file.